BugFixes/README.md

```diff
@@ -33,7 +29,7 @@ Fix 4: Avoid illegal IO access (-4) when GC encounters NIL pointers
 
 As discussed on the mailing list (`PO: Illegal memory access in GC`), the garbage collector
 will read from IO address -4 when it encounters a NIL pointer and tries to read the metadata
-preceding the pointer. By default, IO address -4 is unused and will return 0, but better
+preceding the pointer. By definition, IO address -4 is unused and will return 0, but better
```

Comment:
I will have to reword this paragraph anyway. IO address space is board/hardware dependent and no definition. I was referring to the configuration on the original board and/or compatible ones.

Reply:
Cherry picking :-)
All of those small change proposals are not really necessary. If one or all don't fit, just drop them.